packages/block-library/src/tag-cloud/index.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Server-side rendering of the `core/tag-cloud` block.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Renders the `core/tag-cloud` block on server.
+ *
+ * @param array $attributes The block attributes.
+ *
+ * @return string Returns the tag cloud for selected taxonomy.
+ */
+function render_block_core_tag_cloud( $attributes ) {
+	$class          = "wp-block-tag-cloud align{$attributes['align']}";
+	$wrapper_markup = '<p class="%1$s">%2$s</p>';
+
+	$args = array(
+		'echo'       => false,
+		'taxonomy'   => $attributes['taxonomy'],
+		'show_count' => ! empty( $attributes['showTagCounts'] ),
+	);
+
+	$tag_cloud = wp_tag_cloud( $args );
+
+	$block_content = sprintf(
+		$wrapper_markup,
```

Comment:
nit: I think this would read clearer if we inlined `$wrapper_markup`. It aids with readability when variables are defined near to where they are used.

```php
$block_content = sprintf(
	'<p class="%1$s">%2$s</p>',
	esc_attr( $class ),
	$tag_cloud
);
```